CreateRspecIssue/CreateRspecIssue.ts

```diff
@@ -3,12 +3,13 @@ import { OctokitAction } from "../lib/OctokitAction";
 class CreateRspecIssue extends OctokitAction {
     protected async execute(): Promise<void> {
         this.log("Creating issue");
-        const issue = await this.rest.issues.create(this.addRepo({
+        const issue = (await this.rest.issues.create(this.addRepo({
```

Comment:
You can use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):

```javascript
const { data } = await this.rest.issues.create(this.addRepo({
            title: `Update RSPEC before ${this.payload.milestone.title} release`,
            milestone: this.payload.milestone.number,
            labels: ["Type: Tooling"]
}));
this.log(`Created issue #${data.number}: ${data.html_url}`);
this.createCardIssue(data, this.getInputNumber("column-id"));

Reply:
I used the `(await ... ).data` so the variable name is `issue`. The `data` is not that clear about what it really carries

Reply:
you can use `{ data: issue }` to assign the `data` prop to the var `issue`.

Reply:
Ok, that sounds good (for the next PR)